static/_redirects

```diff
@@ -0,0 +1 @@
+/eggheadio      https://egghead.io/instructors/joe-previte
```

Comment:
In a `_redirects` file, each redirect rule must be listed on a separate line, with the original path followed by the new path or URL.

Reply:
Got i! Thanks for adding that tip!